recipe/meta.yaml

```diff
@@ -1,48 +1,45 @@
 {% set name = "libnvjitlink" %}
-{% set version = "12.4.127" %}
-{% set cuda_version = "12.4" %}
+{% set version = "12.8.93" %}
+{% set cuda_version = "12.8" %}
+{% set platform = "placeholder" %}  # [osx]
 {% set platform = "linux-x86_64" %}  # [linux64]
-{% set platform = "linux-ppc64le" %}  # [ppc64le]
```

Comment:
if you add the ppc stuff back in, and also use `skip: true  # [osx or ppc64le]`, and do the lief fix in the docker container (see nvcc-impl), and remove the binary_relocation, you should be able to sync this fully with conda-forge

Reply:
Thanks. I'll wait for the fix.